cas/incident-reporting.md

```diff
@@ -0,0 +1,101 @@
+
+6. **Engage with the Community**:
+   - Participate in Web PKI forums and email distributions (e.g., public@ccadb.org) to both learn from and contribute to community knowledge.
+   - Share incident learnings in a way that respects confidentiality but helps other organizations prevent similar issues.
+   - Read and adopt best practices found in the [Bugzilla Incident Reports filed by other CA Owners](https://bugzilla.mozilla.org/buglist.cgi?product=CA%20Program&component=CA%20Certificate%20Compliance&bug_status=__open__&list_id=17075089).  
+
+7. **Share Iterative Updates**:
+   - Provide an initial response, followed by regular updates as new information becomes available or as fixes are deployed.
+   - Close the loop with a final summary once the issue is fully resolved, detailing the lessons learned and future prevention strategies.
+  
+8. **Respect External Reporter Privacy**:
+   - If notified of an incident by an external, third party reporter, please respect their privacy by *only* disclosing their name if affirmatively approved to do so (e.g. say "we received a report from a community member" instead of explicitly naming individuals).
+
+###  Incident Reporting: Bad Practices
+
+1. **Generic or Evasive Responses**:
+   - Avoid generic statements that do not address specific issues raised in the report or in response to community questions or comments.
+   - Refrain from providing non-committal or ambiguous answers that might be interpreted as evasive.
+
+2. **Posturing**:
+   - Avoid blaming others or external factors.
+   - Resist downplaying the severity or implications of the incident.
+
+3. **Claims that are Subjective, Unqualified Opinions, Speculative, or Impossible to Substantiate**:
+   - Avoid making claims that are speculative or that cannot be corroborated (e.g. "there is no security impact due to this issue.")
+
+4. **Non-acknowledgment of Responsibility**:
+   - Clearly accept responsibility where and when due, which helps in rebuilding trust and demonstrating accountability.
+
+5. **Superficial Root Cause Analysis**:
+   - Avoid superficial analyses that do not thoroughly explore all contributing factors.
+   - Ensure that the analysis is not prematurely concluded, missing deeper systemic issues.
+
+6. **Committing to Opaque Actions**:
+   - Ensure actions and follow-up work items are detailed and specific.
+   - Promote accountability by describing how the success of an action item can be evaluated as having been successful in addressing the root cause.
+
+7. **Inadequate Monitoring Post-Incident**:
+   - Establish mechanisms to monitor the long-term effectiveness of implemented changes.
+   - Be ready to revisit and revise solutions if subsequent issues indicate that the initial response was not entirely effective.
+  
+## Useful References:
+
+- Root Cause and Incident Analysis Methodologies:
```

Comment:
Love me some Farnam Street and the work of elite fence operator Mr Chesterton! I think a sentence or two about each reference and what we hope the reader to take away from it would make them more valuable, and likely to be read.

Reply:
Summaries are now included in #187.